context.go

```diff
@@ -222,7 +223,18 @@ func (c *Ctx) detectGOPATH(path string) (string, error) {
 // ImportForAbs returns the import path for an absolute project path by trimming the
 // `$GOPATH/src/` prefix.  Returns an error for paths equal to, or without this prefix.
 func (c *Ctx) ImportForAbs(path string) (string, error) {
-	srcprefix := filepath.Join(c.GOPATH, "src") + string(filepath.Separator)
+
+	gopathEvaluated, err := filepath.EvalSymlinks(c.GOPATH)
+	if err != nil {
+		return "", fmt.Errorf("Error evaluating symlinks in GOPATH %s: %s", c.GOPATH, err.Error())
+	}
+
+	path, err = filepath.EvalSymlinks(path)
+	if err != nil {
+		return "", fmt.Errorf("Error evaluating symlinks in %s: %s", path, err.Error())
```

Comment:
Is the intention to show the original path in the error? This looks like ```path``` would be set to the result of ```EvalSymlinks```.